homeassistant/components/tedee/lock.py

```diff
@@ -87,7 +92,11 @@ def is_jammed(self) -> bool:
     @property
     def available(self) -> bool:
         """Return True if entity is available."""
-        return super().available and self._lock.is_connected
+        return (
+            super().available
+            and self._lock.is_connected
+            and self._lock.state != TedeeLockState.UNCALIBRATED
```

Comment:
I would only do that, if we added a separate available property to the bridge, to be able to recognize if the bridge went offline or something is wrong with the lock itself

Reply:
Or we  just raise errors on service calls

Reply:
Can you connect multiple locks to one bridge? If yes we should create a own device for the bridge and connect the lock with the via_device to it.

If the lock is in the uncalibrated state, we don't know anything about the lock state, therefore only raising errors on the service call is wrong in my opinion

Reply:
yes you can. The bridge is already its own device and connected through `via_device`.  Hm, it's only the lock entity so I guess that argument makes sense.

Reply:
In another PR, you could add a diagnostic connection binary entity to the gateway itself